docs/en/query_language/system.md

```diff
@@ -1,9 +1,58 @@
 # SYSTEM Queries {#query_language-system}
 
+- [RELOAD DICTIONARIES](#query_language-system-reload-dictionaries)
+- [RELOAD DICTIONARY](#query_language-system-reload-dictionary)
+- [DROP DNS CACHE](#query_language-system-drop-dns-cache)
+- [DROP MARKS CACHE](#query_language-system-drop-marks-cache)
+- [FLUSH LOGS](#query_language-system-flush_logs)
+- [RELOAD CONFIG](#query_language-system-reload-config)
+- [SHUTDOWN](#query_language-system-shutdown)
+- [KILL](#query_language-system-kill)
 - [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends)
 - [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
 - [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)
 
+## RELOAD DICTIONARIES {#query_language-system-reload-dictionaries}
+
+Reloads all dictionaries that have been successfully downloaded before.
+By default, lazy loading is enabled [dictionaries_lazy_load](../operations/server_settings/settings.md#dictionaries-lazy-load), so dictionaries are not loaded automatically at startup, but only when they first access through dictGet or SELECT to ENGINE = Dictionary. `system reload dictionaries` command reloads such dictionaries (LOADED).
+Always returns `Ok.` regardless of the result of the dictionary update.
+
+## RELOAD DICTIONARY dictionary_name {#query_language-system-reload-dictionary}
+
+Completely reloads a dictionary `dictionary_name`, regardless of the state of the dictionary (LOADED / NOT_LOADED / FAILED).
+Always returns `Ok.` regardless of the result of updating the dictionary.
+The status of the dictionary can be checked by querying `system.dictionaries`.
+
+```sql
+select name,status from system.dictionaries
+```
+
+## DROP DNS CACHE {#query_language-system-drop-dns-cache}
+
+Resets ClickHouse's internal DNS cache. Sometimes it is necessary to use this command when changing the infrastructure (changing the IP address of another ClickHouse server or the server used by dictionaries).
+
+For more convenient (automatic) cache management, see disable_internal_dns_cache, dns_cache_update_period parameters.
```

Comment:
links?

Reply:
They are not documented yet. It's sort of todo for myself.